src/main/java/com/codeborne/selenide/Config.java

```diff
@@ -14,6 +14,8 @@
   String browserBinary();
   String pageLoadStrategy();
   DesiredCapabilities browserCapabilities();
+  String chromeoptionsArgs();
+  String chromeoptionsPrefs();
```

Comment:
I don't like adding the settings. We have too lot of them. And these are chrome-specific settings in the global `Config` class...

If somebody wants to tune Chrome settings so much, he can just implement `WebDriverProvider`, right?